corehq/util/model_log.py

```diff
@@ -1,45 +1,56 @@
 from enum import Enum
 
-from django.contrib.admin.models import ADDITION, CHANGE, DELETION, LogEntry
-from django.contrib.admin.options import get_content_type_for_model
-from django.utils.encoding import force_text
+from corehq.apps.users.models_sql import HQLogEntry, UpdateDetails
+
+MODEL_PROPERTIES_TO_IGNORE = ['_id', '_rev', 'reporting_metadata', 'password']
```

Comment:
add `devices` and `last_device` and `device_ids`

Reply:
https://github.com/dimagi/commcare-hq/pull/29805/commits/faa0d255a0a42b791a8efd11e3c14efe2c926ec9